src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -294,9 +294,6 @@ const StudentApplication = ({ userProfile, authentication }) => {
           (applicant) => applicant.Profile.AD_Username === profile.AD_Username,
         )
       ) {
-        if (unsavedChanges) {
-          saveApartmentApplication(applicationDetails);
-        }
```

Comment:
Where in the editor change process do we save instead?

Reply:
Look at `handleChangeEditorAccepted`

Reply:
In `handleChangeEditorAccepted()` it does `saveApartmentApplication({ ...applicationDetails, EditorProfile: newEditorProfile })`
(that may be approximate, I am guesstimating it from memory)

Reply:
Which is also a good policy, because then it will save after the user clicks "accept", rather than silently saving when the star icon is clicked

Reply:
Oh, yes, agreed.